File: src/main/java/ru/romangr/lolbot/handler/MessagePreprocessor.java
package ru.romangr.lolbot.handler;

import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
public class MessagePreprocessor {

    private final String botName;

    String process(String messageText) {
        return messageText.replace(botName, "").trim();
    }

}

File: src/main/java/ru/romangr/catbot/handler/MessagePreprocessor.kt
package ru.romangr.catbot.handler

import ru.romangr.catbot.telegram.model.Message

class MessagePreprocessor(private val botName: String) {

    fun process(message: Message): Message {
        if (message.text == null) {
            return message
        }
        val processedText = message.text.replace(botName, "").trim()
        return Message(message.id, message.from, message.chat, processedText, message.video)
    }

}
